### PR TITLE
Circleci integration

### PR DESCRIPTION
**Pull request by NWeber-SL**
and updates to other integration articles

### CODE REVIEW

File: using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md [hunk 1]
```diff
@@ -1,40 +1,43 @@
 # Integrating with Bamboo
 
-This section describes how to integrate Bamboo with ShiftLeft.
+Bamboo is a continuous integration and deployment tool that ties automated builds, tests and releases together in a single workflow. You can integrate your Bamboo project with ShiftLeft Inspect for automated code analysis. To do so, configure each Bamboo project to execute the ShiftLeft Inspect shell command [`sl analyze` (or `sl analyze --cpg)`](../inspect/analyzing-applications.md).
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md]: We don't need to specify `sl analyze` variations - there are many flags we're not specifying anyway.

```suggestion
Bamboo is a continuous integration and deployment tool that ties automated builds, tests and releases together in a single workflow. You can integrate your Bamboo project with ShiftLeft Inspect for automated code analysis. To do so, configure each Bamboo project to execute the ShiftLeft Inspect shell command [`sl analyze`](../inspect/analyzing-applications.md).
```

File: using-inspect-protect/integrating-with-shiftleft/integrating-circleci.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Integrating with CircleCI
+
+CircleCI automates your software builds, tests and deployments, enabling a CI/CD development process. You can integrate your CircleCI project with ShiftLeft Inspect for automated code analysis. To do so, configure each CircleCI project to execute the ShiftLeft Inspect shell command [`sl analyze` (or `sl analyze --cpg)`](../inspect/analyzing-applications.md).
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/integrating-with-shiftleft/integrating-circleci.md]:
```suggestion
CircleCI automates your software builds, tests and deployments, enabling a CI/CD development process. You can integrate your CircleCI project with ShiftLeft Inspect for automated code analysis. To do so, configure each CircleCI project to execute the ShiftLeft Inspect shell command [`sl analyze`](../inspect/analyzing-applications.md).
```

File: using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md (hunk 1 above)
Comment (NWeber-SL) [reply using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md]: Removed in all articles

File: using-inspect-protect/integrating-with-shiftleft/integrating-circleci.md (hunk 2 above)
Comment (NWeber-SL) [reply using-inspect-protect/integrating-with-shiftleft/integrating-circleci.md]: Removed

**Pull request merged**